docs/scos/dev/sdk/zed-api/zed-api.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Zed API
+description: This section lists documents inside the Zed API section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The *Zed API* section includes the following documents:
```

Comment:
Can you add a few words on what this section is about?

Reply:
```suggestion
Explore the flexible REST API offered by Spryker OS for Zed and the basic functionality of mapping REST actions to the backend persistence layer. Learn how to configure the API and explore the Zed API processor stack and the project implementation process for activating API and API bundles. Also, expand your knowledge about Zed API resources to leverage their capabilities effectively.

For more detailed information, see the following documents:
```